iothub/device/src/Transport/Amqp/IAmqpConnectionHolder.cs

```diff
@@ -1,23 +1,16 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.Azure.Devices.Client.Transport.AmqpIoT;
 using System;
 using System.Threading.Tasks;
-using Microsoft.Azure.Devices.Client.Transport.AmqpIoT;
-using Microsoft.Azure.Devices.Shared;
 
 namespace Microsoft.Azure.Devices.Client.Transport.Amqp
 {
-    internal interface IAmqpConnectionHolder
+    internal interface IAmqpConnectionHolder : IDisposable
```

Comment:
This is getting very confusing very fast. Please remove these interfaces.

Reply:
Table P0